From: Ole Streicher <olebole@debian.org>
Date: Wed, 4 Oct 2023 10:46:09 +0200
Subject: Don't fail on passed tests that were marked as xfail

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 974f000..f8c8298 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -185,7 +185,6 @@ convention = 'numpy'
 minversion = "6.0"
 addopts = "-ra  --strict-config  --strict-markers"
 log_cli_level = "INFO"
-xfail_strict = true
 testpaths = ["skimage", "benchmarks"]
 python_files = ["benchmark_*.py", "test_*.py"]
 python_classes = ["Test*", "*Suite"]
