From: Yangfl <mmyangfl@gmail.com>
Date: Wed, 17 Oct 2018 14:04:03 +0800
Subject: Don't download intersphinx mapping

---
 doc/source/conf.py | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 20bbbab..b32e78e 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -243,12 +243,15 @@ numpydoc_class_members_toctree = False
 
 # -- intersphinx --------------------------------------------------------------
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3/", None),
-    "numpy": ("https://numpy.org/doc/stable/", None),
-    "neps": ("https://numpy.org/neps/", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
-    "sklearn": ("https://scikit-learn.org/stable/", None),
-    "matplotlib": ("https://matplotlib.org/stable/", None),
+    'python': ("https://docs.python.org/3/", '/usr/share/doc/python3.9/html/objects.inv'),
+    'numpy': ('https://numpy.org/doc/stable',
+              './_intersphinx/numpy-objects.inv'),
+    'scipy': ('https://docs.scipy.org/doc/scipy/',
+              './_intersphinx/scipy-objects.inv'),
+    'sklearn': ('https://scikit-learn.org/stable',
+                './_intersphinx/sklearn-objects.inv'),
+    'matplotlib': ('https://matplotlib.org/',
+                   (None, './_intersphinx/matplotlib-objects.inv'))
 }
 
 # -- Source code links -------------------------------------------------------
