From: Ole Streicher <olebole@debian.org>
Date: Thu, 11 Apr 2024 14:16:41 +0200
Subject: Don't turn deprecation warnings into errors

---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index 53de334..d6aaa68 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -196,6 +196,7 @@ python_classes = ["Test*", "*Suite"]
 python_functions = ["time_*", "test_*", "peakmem_*"]
 filterwarnings = [
     "error",
+    "ignore::DeprecationWarning",
     'default:.*pyamg, which cannot \(yet\) be imported with NumPy >=2:RuntimeWarning'
 ]
 
