From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Mon, 16 Aug 2021 21:19:05 -0400
Subject: Enable pytest skipping at module level

---
 skimage/_shared/testing.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/skimage/_shared/testing.py b/skimage/_shared/testing.py
index dbdccd6..67e5e48 100644
--- a/skimage/_shared/testing.py
+++ b/skimage/_shared/testing.py
@@ -209,13 +209,13 @@ def teardown_test():
     """
     warnings.simplefilter('default')
 
-
 def fetch(data_filename):
     """Attempt to fetch data, but if unavailable, skip the tests."""
     try:
         return data._fetch(data_filename)
     except (ConnectionError, ModuleNotFoundError):
-        pytest.skip(f'Unable to download {data_filename}')
+        pytest.skip(f'Unable to download {data_filename}',
+                    allow_module_level=True)
 
 
 def test_parallel(num_threads=2, warnings_matching=None):
