From: Ole Streicher <olebole@debian.org>
Date: Mon, 29 Jun 2020 16:52:21 +0200
Subject: Include everyting from the data subdir

---
 setup.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/setup.cfg b/setup.cfg
index 95434b3..74fb0f7 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -6,3 +6,6 @@ python_functions=time_* test_* peakmem_*
 omit= */tests/*
 [metadata]
 license_file = LICENSE.txt
+
+[options.package_data]
+skimage = data/*
