From: Ole Streicher <olebole@debian.org>
Date: Mon, 11 Dec 2023 11:38:35 +0100
Subject: Remove sphinx_gallery dependency

---
 doc/source/conf.py | 30 +-----------------------------
 1 file changed, 1 insertion(+), 29 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 08d439b..4fa9039 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -13,8 +13,6 @@ import plotly.io as pio
 import skimage
 from packaging.version import parse
 from plotly.io._sg_scraper import plotly_sg_scraper
-from sphinx_gallery.sorting import ExplicitOrder
-from sphinx_gallery.utils import _has_optipng
 
 filterwarnings(
     "ignore", message="Matplotlib is currently using agg", category=UserWarning
@@ -49,7 +47,6 @@ extensions = [
     "sphinx.ext.linkcode",
     "sphinx.ext.mathjax",
     "sphinx_copybutton",
-    "sphinx_gallery.gen_gallery",
     "doi_role",
     "numpydoc",
     "sphinx_design",
@@ -92,34 +89,9 @@ else:
 
 pio.renderers.default = "sphinx_gallery_png"
 
-sphinx_gallery_conf = {
-    "doc_module": ("skimage",),
-    "examples_dirs": "../examples",
-    "gallery_dirs": "auto_examples",
-    "backreferences_dir": "api",
-    "reference_url": {"skimage": None},
-    "image_scrapers": ("matplotlib", plotly_sg_scraper),
-    "subsection_order": ExplicitOrder(
-        [
-            "../examples/data",
-            "../examples/numpy_operations",
-            "../examples/color_exposure",
-            "../examples/edges",
-            "../examples/transform",
-            "../examples/registration",
-            "../examples/filters",
-            "../examples/features_detection",
-            "../examples/segmentation",
-            "../examples/applications",
-            "../examples/developers",
-        ]
-    ),
-    # Remove sphinx_gallery_thumbnail_number from generated files
-    "remove_config_comments": True,
-}
 
 
-if _has_optipng():
+if False:
     # This option requires optipng to compress images
     # Optimization level between 0-7
     # sphinx-gallery default: -o7
