From: Ole Streicher <olebole@debian.org>
Date: Mon, 11 Dec 2023 11:38:35 +0100
Subject: Remove sphinx_gallery dependency

---
 doc/source/conf.py | 34 +---------------------------------
 1 file changed, 1 insertion(+), 33 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index efcd864..117c0bc 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -14,8 +14,6 @@ import skimage
 from intersphinx_registry import get_intersphinx_mapping
 from packaging.version import parse
 from plotly.io._sg_scraper import plotly_sg_scraper
-from sphinx_gallery.sorting import ExplicitOrder
-from sphinx_gallery.utils import _has_optipng
 
 filterwarnings(
     "ignore", message="Matplotlib is currently using agg", category=UserWarning
@@ -50,7 +48,6 @@ extensions = [
     "sphinx.ext.linkcode",
     "sphinx.ext.mathjax",
     "sphinx_copybutton",
-    "sphinx_gallery.gen_gallery",
     "doi_role",
     "numpydoc",
     "sphinx_design",
@@ -93,38 +90,9 @@ else:
 
 pio.renderers.default = "sphinx_gallery_png"
 
-sphinx_gallery_conf = {
-    "doc_module": ("skimage",),
-    "examples_dirs": "../examples",
-    "gallery_dirs": "auto_examples",
-    "backreferences_dir": "api",
-    "reference_url": {"skimage": None},
-    "image_scrapers": ("matplotlib", plotly_sg_scraper),
-    "subsection_order": ExplicitOrder(
-        [
-            "../examples/data",
-            "../examples/numpy_operations",
-            "../examples/color_exposure",
-            "../examples/edges",
-            "../examples/transform",
-            "../examples/registration",
-            "../examples/filters",
-            "../examples/features_detection",
-            "../examples/segmentation",
-            "../examples/applications",
-            "../examples/developers",
-        ]
-    ),
-    # Remove sphinx_gallery_thumbnail_number from generated files
-    "remove_config_comments": True,
-    # `True` defaults to the number of jobs used by Sphinx (see its flag `-j`)
-    #   Temporarily disabled because plotly scraper isn't parallel-safe
-    #   (see https://github.com/plotly/plotly.py/issues/4959)!
-    # "parallel": True,
-}
 
 
-if _has_optipng():
+if False:
     # This option requires optipng to compress images
     # Optimization level between 0-7
     # sphinx-gallery default: -o7
